src/DotNetWorker.Grpc/IGrpcRequestHandler.cs

```diff
@@ -0,0 +1,13 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+
+using Microsoft.Azure.Functions.Worker.Grpc.Messages;
+using System.Threading.Tasks;
+
+namespace Microsoft.Azure.Functions.Worker.Grpc
+{
+    internal interface IMessageProcessor
+    {
+        Task ProcessMessageAsync(StreamingMessage request);
```

Comment:
Do we want `CancellationToken` on these APIs? Or is the cancellation inside `StreamingMessage`?

Same question for `IWorkerClient.SendMessageAsync`.

Reply:
I think it makes sense for these interfaces to take in a `CancellationToken`. Will update